tasks/prerequisites.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Install system prerequisites
+  apt:
```

Comment:
In my opinion we should use 'package' instead of 'apt'

Reply:
Are you sure that the same package names work?